data/cfp.md

```diff
@@ -0,0 +1,99 @@
+## DACHFest 2018 Call for Papers
+
+Thanks for your interest in speaking at DACHFest 2018!
+
+DACHFest is a community-driven conference organized by Google Developer Groups from Germany, Austria, and Switzerland. It's taking place at the Munich Technical University on November 10–11: around 800 people are going to get together to enjoy 3 talk tracks, a lot of community-oriented activities, and opportunities for networking and making new friends. ❤️
+
+If you have any questions, do not hesitate to contact us on [Twitter](https://twitter.com/DACHFest) or at [cfp@dachfest.com](mailto:cfp@dachfest.com).
+
+### Deadlines 📅
+
+September 15: Last day to submit your application!
+
+September 30: We'll notify you of your submission status.
+
+### Topics 💭
+
+DACHFest's main theme is _making a difference_. We're not limiting submissions to any particular area or stack — we just want to hear about that thing you did that touched people's lives!
+
+Are you helping your community through tech? Tell us about it!
+
+Have you created something that changes the way people work? We're waiting for you!
+
+You want to tell our participants something that will make a difference in _their_ lives? Please submit right now!
+
+And don't forget that the examples above are just very rough guidelines. If you're not sure whether your talk is a good fit, stop hesitating: it probably is! Surprise us!
+
+P.S. If you have more questions about the theme, suggested topics, or the right way to frame your talk, you can always get in touch with our wonderful mentors — more about them in a bit! — or ask [@dachfest](https://twitter.com/dachfest) on Twitter.
+
+### Selection Process ✅
+
+Our diverse program committee comprises both generalists and specialists, and we will try our best to make sure that every submitted talk, no matter how narrow, is reviewed by recognized experts from the same field.
+
+We strive for eliminating bias as much as possible, and it means that every submission will be anonymized and evaluated entirely on its merits. We are not looking for hot startups and famous names — we are looking for exciting stories and interesting people.
+
+No matter whether your talk is selected or not, we will always try to provide feedback and give you suggestions if we have any.
+
+### Guidelines 📃
+
+* **Language**: English.
+* **Brevity**: we recommend a single paragraph for the abstract and 3-4 paragraphs for the talk description. Get straight to the point!
+* **Formatting and proofreading**: submissions with good spelling, no typos, and proper paragraph formatting make the job of our program committee much easier.
+* **Completeness**: you will not have a chance to modify your submission! Read it carefully before sending, make sure it's complete and has all the information we need to make an objective decision.
```

Comment:
+ Ask someone to review your abstract before submission. Our mentors are also happy to guide you and give valuable advice for your entry.

Reply:
i see now that it is below. but maybe it's worth mentioning here as well

Reply:
Done